src/components/Forms/SubscribeForm.js

```diff
@@ -132,18 +124,14 @@ class SubscribeForm extends React.Component {
             onSubmit={values => this.handleSubmit(values)}
             render={({ errors }) => (
               <Form
-                css={css({
-                  display: 'flex',
-                  flexDirection: 'column',
-                  alignItems: 'flex-end',
-                  marginTop: '20px',
-                  '.field-error': {
-                    color: theme.colors.red,
-                  },
-                  [bpMinSM]: {
-                    flexDirection: 'row',
+                css={css(
+                  {
+                    '.field-error': {
+                      color: colors.red,
```

Comment:
I notice that colors come in theme and from this colors file. Should they all be in the theme? Just wondering what the difference is. Was also thinking fonts might be part of the theme but I don't know.

Reply:
My first comment is about it ☝️

Reply:
I understand the "theme" like something that contains the things that should change. So I think keeping const stuff like colors (base colors!), fonts etc in separate files is a good idea. Makes sense?

Reply:
Also I think that we can keep one `lib.js` file that exports this constant things, so we can export it like `import { colors, fonts, ...}` VS two lines of import from separate files.